frontend/model/notifications/utils.js

```diff
@@ -1,6 +1,6 @@
 import type { Notification } from './types.flow.js'
 
-import { notificationStorageLimits } from '@model/database.js'
+import { notificationStorageLimits } from '~/frontend/model/database.js'
```

Comment:
Yea, if I remember correctly I had to change `@model/` to `~/frontend/` because the former caused an error.

Reply:
Now, I can understand what you mean. I have just tracked the Travis error which is very much similar to your case.
Travis fails because of the absolute path. I converted it into a related path and it seems working. Weird Travis...